ui-tests/src/test/resources/features/other/data-virtualization.feature

```diff
@@ -0,0 +1,76 @@
+# @sustainer: mmajerni@redhat.com
+@dv
+Feature: Data Virtualization smoke tests
+
+  Background:
+    Given clean application state
+    And deploy DV
+    And log into the Syndesis
+    And navigate to the "Data" page
+
+  @dvSmoke
+  Scenario: basic smoke ui test
+    When click on the "Create Data Virtualization" link to create a new data virtualization
+    And fill in values by element ID
+      | virtname        | smokeVirtualization              |
+      | virtdescription | smoke virtualization description |
+
+    And click on the "Create" button
+    And sleep for 1 seconds
+    Then check that data virtualization "smokeVirtualization" is present in virtualizations list
+    And check that data virtualization "smokeVirtualization" has description "smoke virtualization description"
+
+    When edit data virtualization "smokeVirtualization"
+    And click on the "Create a view" link
+    And select DV connection "PostgresDB" on "create" page
+    And select DV connection tables on create page
+      | contact |
+
+    And click on the "Next" link
+    And fill in values by element ID
+      | name        | testView             |
+      | description | testView description |
+    And click on the "Done" button
+    And click on the "Done" button
+
+    Then check that data virtualization view "testView" is present in view list
+
+    When click on the "Import views" link
+    And select DV connection "PostgresDB" on "import" page
+    And click on the "Next" link
+    And select DV connection tables on import page
+      | contact |
+      | todo    |
+    And click on the "Done" button
+
+    Then check that data virtualization view "contact" is present in view list
+    And check that data virtualization view "todo" is present in view list
+
+    When go to "SQL client" tab on virtualization page
+    And set parameters for SQL client
+      | contact | 10 | 2 |
+    And click on the "Submit" button
+    Then check that number of rows for query is 0 in sql client
```

Comment:
I would suggest checking whether the SQL client shows any data without offset and after that test whether offset works. Also, you should not rely on data which is in the `contact` table by default ` Joe | Jackson` . I suggest to add this into the background phase:
```
    And reset content of "contact" table
    And inserts into "contact" table
      | Joe | Jackson | Red Hat | db |
```

Reply:
Added one row to contact table and five to todo table. Checking contact table without offset and todo table with offset.